.werft/util/slack.js

```diff
@@ -0,0 +1,51 @@
+const https = require('https');
+
+function sendToSlack(context, onErr) {
```

Comment:
this is way too generic a name for a function that does something very specific: inform about build failures